### PR TITLE
Created manifest for Datto Backup Agent 3.0.3.0

### PR DESCRIPTION
**Pull request by seejdev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34001)

### CODE REVIEW

File: manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Datto.BackupAgent
+PackageVersion: 3.0.3.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+  Custom: /norestart
```
Comment (Trenly) [inline manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.installer.yaml:11]:
```suggestion
  Custom: /norestart
InstallerSuccessCodes:
- 3010
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: Datto.BackupAgent
+PackageVersion: 3.0.3.0
+PackageLocale: en-US
+Publisher: Datto Inc.
+PublisherUrl: https://www.datto.com
+PublisherSupportUrl: https://www.datto.com/support
+# PrivacyUrl: 
+Author: Datto Inc.
+PackageName: Datto Backup Agent
```
Comment (ItzLevvie) [inline manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.locale.en-US.yaml:12]:
```suggestion
PackageName: Datto Windows Agent
```

File: manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+PackageUrl: https://www.datto.com
+License: Proprietary
+LicenseUrl: https://www.datto.com/legal/agent-licensing-agreement
+Copyright: Copyright (c) Microsoft Corporation
```
Comment (ItzLevvie) [inline manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.locale.en-US.yaml:16]:
```suggestion
Copyright: Copyright (c) Datto Inc.
```

File: manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+ProductCode: '{3862CAAC-9FF6-409C-8F29-49FE503405C9}'
```
Comment (ItzLevvie) [inline manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.installer.yaml:15]:
```suggestion
ProductCode: '{910c4dd5-1519-4423-8292-7f4cb168617b}'
```
`winget uninstall "{3862CAAC-9FF6-409C-8F29-49FE503405C9}"` does not work

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/34001#pullrequestreview-800141906

**Pull request closed without merging**